native/ios/Integreat/Info.plist

```diff
@@ -66,6 +66,41 @@
 	<string>Integreat wants to use your location to find nearby cities.</string>
 	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
 	<string>Integreat wants to use your location to find nearby cities.</string>
+	<key>NSPrivacyAccessedAPITypes</key>
+		<array>
+			<dict>
+				<key>NSPrivacyAccessedAPIType</key>
+				<string>NSPrivacyAccessedAPICategoryDiskSpace</string>
+				<key>NSPrivacyAccessedAPITypeReasons</key>
+				<array>
+					<string>E174.1</string>
```

Comment:
Okay all of these are really hard...
I think this might be a problem:
> The app must behave differently based on disk space in a way that is observable to users.

I think this is not the case for us (except perhaps the app crashing or not working if there is no space anymore). However, I still agree that this is probably the right category.

Reply:
i mean its requested. probably due to writing files the libraries somehow use this before writing on local device?!